src/NServiceBus.Persistence.CosmosDB.AzureStorageSagaExporter/ToolVersion.cs

```diff
@@ -41,7 +41,15 @@ public static async Task<bool> CheckIsLatestVersion(ILogger logger, bool ignoreU
 
                 if (latest > current)
                 {
-                    void Log(string message)
+                    var packageVersion = latest.ToNormalizedString();
+
+                    log($"*** New version detected: {packageVersion}");
+                    log("*** Update to the latest version using the following command:");
+                    log($"***   dotnet tool update --tool-path <installation-path> {PackageID} --add-source {FeedUri} --version {packageVersion}");
+
+                    return ignoreUpdates;
+
+                    void log(string message)
```

Comment:
Why is a local function lower case? It is like any other method right?

C# convention is that all methods and functions name should be in PascalCase, so I would go with standard convention regardless of their accessibility

Reply:
Unlike other method names, it's a local function. Microsoft doesn't provide clear guidance on the local functions naming but the samples use both. Also, positioning local functions at the end of the logical scope they belong to. Which is what I've done here.

If you think local functions should be starting with an upper case or we'll have a StyleCop rule that will enforce, happy to change.

Reply:
I am going to approve and merge as-is for now. This is something that can be easily changed later if we determine the casing is incorrect.

Reply:
The position is fine. So why do you think I used upper case like regular method names? I can see clear benefits in moving the local function for readability while deviating from regular method and function name convention doesn't seem to give value. Anyway not worth continuing the discussion.